api/comments.rb

```diff
@@ -62,5 +62,4 @@
       pass
     end
   end
-  comment.to_hash.to_json
```

Comment:
@dianakhuang and I agree, but it seems that there are details from the previously existing comment that are used in edx-platform.

So, either we need to make a separate call before the delete to gather that data, or we need to dig further to determine how and if there is a way to clean up the edx-platform code such that it doesn't require this result.

@dianakhuang and I voted to leave this in and move on, but I was wavering, cause I don't like that this is here and doesn't make logical sense.